server/webhook/issue_fixture_test.go

```diff
@@ -0,0 +1,510 @@
+		"visibility_level":20,
+		"path_with_namespace":"manland/webhook",
+		"default_branch":"master",
+		"ci_config_path":null,
+		"homepage":"http://localhost:3000/manland/webhook",
+		"url":"ssh://rmaneschi@localhost:2222/manland/webhook.git",
+		"ssh_url":"ssh://rmaneschi@localhost:2222/manland/webhook.git",
+		"http_url":"http://localhost:3000/manland/webhook.git"
+	},
+	"object_attributes":{
+		"author_id":1,
+		"closed_at":null,
+		"confidential":false,
+		"created_at":"2019-04-06 21:03:04 UTC",
+		"description":"hello world!",
+		"due_date":null,
+		"id":181,
+		"iid":1,
+		"last_edited_at":null,
+		"last_edited_by_id":null,
+		"milestone_id":null,
+		"moved_to_id":null,
+		"project_id":24,
+		"relative_position":1073742323,
+		"state":"opened",
+		"time_estimate":0,
+		"title":"test new issue",
+		"updated_at":"2019-04-06 21:03:04 UTC",
+		"updated_by_id":null,
+		"url":"http://localhost:3000/manland/webhook/issues/1",
+		"total_time_spent":0,
+		"human_total_time_spent":null,
+		"human_time_estimate":null,
+		"assignee_ids":[50],
+		"assignee_id":50,
+		"action":"open"
+	},
+	"labels":[],
+	"changes":{
+		"author_id":{
+			"previous":null,
+			"current":1
+		},
+		"created_at":{
+			"previous":null,
+			"current":"2019-04-06 21:03:04 UTC"
+		},
+		"description":{
+			"previous":null,
+			"current":"hello world!"
+		},
+		"id":{
+			"previous":null,
+			"current":181
+		},
+		"iid":{
+			"previous":null,
+			"current":1
+		},
+		"project_id":{
+			"previous":null,
+			"current":24
+		},
+		"relative_position":{
+			"previous":null,
+			"current":1073742323
+		},
+		"state":{
+			"previous":null,
+			"current":"opened"
+		},
+		"title":{
+			"previous":null,
+			"current":"test new issue"
+		},
+		"updated_at":{
+			"previous":null,
+			"current":"2019-04-06 21:03:04 UTC"
+		},
+		"assignees":{
+			"previous":[],
+			"current":[{
+				"name":"manland",
+				"username":"manland",
+				"avatar_url":"https://www.gravatar.com/avatar/c6b552a4cd47f7cf1701ea5b650cd2e3?s=80\\u0026d=identicon"
+				}]
+			},
+			"total_time_spent":{
+				"previous":null,
+				"current":0
+			}
+		},
+		"repository":{
+			"name":"webhook",
+			"url":"ssh://rmaneschi@localhost:2222/manland/webhook.git",
+			"description":"",
+			"homepage":"http://localhost:3000/manland/webhook"
+		},
+		"assignees":[{
+			"name":"manland",
+			"username":"manland",
+			"avatar_url":"https://www.gravatar.com/avatar/c6b552a4cd47f7cf1701ea5b650cd2e3?s=80\\u0026d=identicon"
+			}]
+			}`
+
+const NewIssueUnassigned = `{
+				"object_kind":"issue",
+				"event_type":"issue",
+				"user":{
+					"name":"Administrator",
+					"username":"root",
+					"avatar_url":"https://www.gravatar.com/avatar/e64c7d89f26bd1972efa854d13d7dd61?s=80\\u0026d=identicon"
+				},
+				"project":{
+					"id":24,
+					"name":"webhook",
+					"description":"",
+					"web_url":"http://localhost:3000/manland/webhook",
+					"avatar_url":null,
+					"git_ssh_url":"ssh://rmaneschi@localhost:2222/manland/webhook.git",
```

Comment:
Pretty sure you want to remove this.

Reply:
I prefer to keep it because its original web hook from gitlab. Its not criticial or sensible info.